runtimes/jsoo/runtime.ml

```diff
@@ -59,13 +59,18 @@ let duration_to_jsoo d =
   end
 
 let date_of_jsoo d =
-  R_ocaml.date_of_numbers d##getUTCFullYear d##getUTCMonth d##getUTCDate
-
-let date_to_jsoo d =
-  let years = R_ocaml.integer_to_int (R_ocaml.year_of_date d) in
-  let months = R_ocaml.integer_to_int (R_ocaml.month_number_of_date d) in
-  let days = R_ocaml.integer_to_int (R_ocaml.day_of_month_of_date d) in
-  new%js Js.date_day years months days
+  let d = Js.to_string d in
+  let d =
+    if String.contains d 'T' then d |> String.split_on_char 'T' |> List.hd
```

Comment:
Ok wouahou you already thought of splitting the time here, kudos.

Reply:
:innocent: